### ISSUE TITLE
Message sent sucessfully from console but could not get receive in app

### ISSUE THREAD

**Issue by harshkumarios** [mesibo/samples#24]
I am using mesibo in ios. Set App delegate methods and created a user in console.
I tried to send message but not receive by device.But the status is message sent sucessfully.

**Comment by Abhishek284** [comment]
I am not able to send message from console. I always get "Message failed to send" even though the user is online. Is there anything other than the steps provided in the tutorials need to taken care to send message from console?

**Comment by mesibo** [comment]
Ensure that the 'from' field is a valid user.

**Comment by Abhishek284** [comment]
Yes, I have made sure that 'from' field is a valid user. I have created two users and made them to be online using two devices and still I get "Message failed to send".

**Comment by Chitengi** [comment]
I am having the same problem too. Kindly clarify how this is achieved from an emulator from Android studio. Also explain what could make a user invalid if the user is created from within the console by ticking the create user checkbox.

**Comment by Chitengi** [comment]
> I am having the same problem too. Kindly clarify how this is achieved from an emulator from Android studio. Also explain what could make a user invalid if the user is created from within the console by ticking the create user checkbox.

I managed to send the message . To a real device

**Comment by mesibo** [comment]
It doesn't matter that the device is emulator or e real-device. Ensure that 'from' is a real user. Also, the destination device is online (connection status is 1). 

If you still have an issue, please email us app id, destination and the from user and we will look into it.

**Comment by Chitengi** [comment]
> It doesn't matter that the device is emulator or e real-device. Ensure that 'from' is a real user. Also, the destination device is online (connection status is 1).
> 
> If you still have an issue, please email us app id, destination and the from user and we will look into it.

I'm trying to send a message from the app and I have noticed that the tutorial says three things are necessary when sending out messages i.e  destination user, message id and message itself yet the provided code's send function only requires two String variables.  I'm getting the following error after providing an address and a message;- I/MesiboJNI: E(2606-201109-268)(11453): *** BUG *** debug_global: missing address and uid.

**Comment by Chitengi** [comment]
Okay, I have seen the third parameter presented as a random Mesibo number.

**Comment by sunidhioodles** [comment]
Hi did anyone figure out how to deal with this issue, I am facing the same thing? The message is sent successfully from the console but is never received on the application.

**Comment by sunidhioodles** [comment]
> > I am having the same problem too. Kindly clarify how this is achieved from an emulator from Android studio. Also explain what could make a user invalid if the user is created from within the console by ticking the create user checkbox.
> 
> I managed to send the message . To a real device

Hi, How did you do that? Could you please help

**Comment by pangia** [comment]
also having issue where message is sent successfully from the console but is never received in the iOS application. Receiving user is online with connection status of 1. message listener never gets called.

**Comment by mesibo** [comment]
Share your app id, `from` and `to` user. You can also send us private ticket at https://mesibo.com/ticket

**Comment by pangia** [comment]
> Share your app id, `from` and `to` user. You can also send us private ticket at https://mesibo.com/ticket

app id 7122. users = 648023 & 11111. 648023 was online. 11111 sent message from console.  Message sent successfully, never hit the listener in app. Thanks. (I tried to submit a ticket, but never received a confirmation email from site when signing up for access).

**Comment by mesibo** [comment]
@pangia, I checked logs and can see that your messages were delivered. Let me know if you still having issues.
